RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -375,6 +375,10 @@ boolean autoEat(int howMany, item toEat, boolean silent)
 		{
 			use(1, $item[whet stone]);
 		}
+		if(item_amount($item[mini kiwi aioli]) > 0) //use mini kiwi aioli if we got one from the mini kiwi
+		{
```

Comment:
It does not. It's similar to the whet stone. We should only be using it though once we are ready to eat. I haven't actualy consumed any in my runs, because the mini kiwi is being used on the digitized cookies, but in a Standard run vs 11THIAU the cookies should be fairly low on the consumables list relative to cookbookbat food

Reply:
Unlike whet stones, mafia doesn't have tracking for how many unused aioli you have used (yet) unfortunately. So using before consuming is the only way right now to make sure they're not being under or over used.